pom.xml

```diff
@@ -135,27 +140,22 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-support</artifactId>
       <version>3.3</version>
-      <exclusions>
-        <!-- -->
-        <exclusion>
-          <groupId>org.jboss.marshalling</groupId>
-          <artifactId>jboss-marshalling</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.jboss.marshalling</groupId>
-          <artifactId>jboss-marshalling-river</artifactId>
-        </exclusion>
-      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
Is this just to update Junit to 4.13.1? It may be better to update the `plugin-pom` to 4.16 https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.16

Reply:
No, it's just an extra instructions. They were required with jenkins specific depend version.
Regarding your suggestion to upgrade  plugin pom to  4.16 will take a look tomorrow